tests/test_simpleflow/test_execute.py

```diff
@@ -297,6 +297,7 @@ def create_sleeper_subprocess():
     return pid
 
 
+@pytest.mark.skipif(platform.system() == 'Darwin', reason="psutil process statuses are buggy on OSX")
```

Comment:
(ignoreme) Maybe xfail, for the day psutil is fixed 🙂

Reply:
Yay. I noticed it broke on pypy too on travis this week-end, but succeeded on the next try. I'll change the "skipif" to "xfails" everywhere, since they are bugs and would be nice to have when psutil gets fixed.